src/collection/collection.service.ts

```diff
@@ -324,6 +326,19 @@ export class CollectionService {
 
 		return omit(collection, propertiesToDelete);
 	}
+	public static async getBundles(limit: number): Promise<Avo.Collection.Collection[]> {
+		try {
+			const response = await dataService.query({
+				query: GET_BUNDLES,
+				variables: { limit },
+			});
+
+			return get(response, 'data.app_collections', []);
+		} catch (err) {
+			console.error('Failed to fetch bundles.', err);
+			throw new CustomError('Het ophalen van de bundels is mislukt.', err);
```

Comment:
const error = new CustomError('Het ophalen van de bundels is mislukt.', err, {query: 'GET_BUNDLES'})
console.error(error)
throw error;